[2024-06-30 19:01:09,987] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: example_datalake_dag.create_datalake_table scheduled__2024-06-30T03:00:00+00:00 [queued]>
[2024-06-30 19:01:10,129] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: example_datalake_dag.create_datalake_table scheduled__2024-06-30T03:00:00+00:00 [queued]>
[2024-06-30 19:01:10,147] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-06-30 19:01:10,152] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2024-06-30 19:01:10,153] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-06-30 19:01:10,364] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): create_datalake_table> on 2024-06-30 03:00:00+00:00
[2024-06-30 19:01:10,523] {standard_task_runner.py:52} INFO - Started process 1310 to run task
[2024-06-30 19:01:10,750] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'example_datalake_dag', 'create_datalake_table', 'scheduled__2024-06-30T03:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpn6gyue4e', '--error-file', '/tmp/tmpxaaknrlt']
[2024-06-30 19:01:11,143] {standard_task_runner.py:80} INFO - Job 11: Subtask create_datalake_table
[2024-06-30 19:01:14,023] {task_command.py:371} INFO - Running <TaskInstance: example_datalake_dag.create_datalake_table scheduled__2024-06-30T03:00:00+00:00 [running]> on host 12d6cc628018
[2024-06-30 19:01:15,579] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_datalake_dag
AIRFLOW_CTX_TASK_ID=create_datalake_table
AIRFLOW_CTX_EXECUTION_DATE=2024-06-30T03:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-06-30T03:00:00+00:00
[2024-06-30 19:01:15,810] {base.py:68} INFO - Using connection ID 'datalake_postgres' for task execution.
[2024-06-30 19:01:15,840] {dbapi.py:231} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS datalake_table (
            id SERIAL PRIMARY KEY,
            data VARCHAR(50)
        );
        , parameters: None
[2024-06-30 19:01:15,857] {postgres.py:94} INFO - NOTICE:  relation "datalake_table" already exists, skipping

[2024-06-30 19:01:17,351] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=example_datalake_dag, task_id=create_datalake_table, execution_date=20240630T030000, start_date=20240630T190110, end_date=20240630T190117
[2024-06-30 19:01:17,843] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-06-30 19:01:19,165] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
